src/lib/OpenEXR/ImfSimd.h

```diff
@@ -46,6 +46,10 @@
 #    define IMF_HAVE_NEON
 #endif
 
+#if defined(__aarch64__)
+#    define IMF_HAVE_NEON_AARCH64 1
```

Comment:
I'm groping in the dark without an AArch64 machine, but don't you mean the `#define IMF_HAVE_NEON_AARCH64` to be inside the `#if defined(__ARM_NEON)` above? Don't you want this only if `__ARM_NEON` _and_ `__aarch64__`? Or does `__aarch64__` sufficient itself?

Reply:
64-bit ARM makes NEON the standard feature that's "always there", so no need for a separate check for it. Very similar to how x64 made SSE2 be guaranteed.

Reply:
Got it, thanks.